velox/common/caching/GroupTracker.cpp

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/caching/GroupTracker.h"
+
+#include "velox/common/base/BitUtil.h"
+
+DEFINE_int32(
```

Comment:
GFlags and global singletons are hard to keep track of and test. Can we add this property to constructor and have some higher-level entity own an instance?